### ISSUE TITLE
About the PyTorch version

### ISSUE THREAD

**Issue by flexibility2** [graykode/DAC#2]
Hi, I am very grateful for your source code! I try to implement this code with pytorch vision, but it does't work. I wonder if you have tried this? Can I communicate with you about this more specifically？ Thanks a lot!

**Comment by graykode** [comment]
Hello. This repository does not support pytorch, only tensorflow

**Comment by graykode** [comment]
Sure How can I help you?

**Comment by flexibility2** [comment]
@graykode , sorry to disturb you, I wonder if you could help me when you are free. This is my implementation:
https://github.com/flexibility2/DAC-Pytorch
The biggest problem in my implementation is that the loss of every batch is close to zero, and after a few iterations, it really becomes zero. I will be very appreciated for your generosity! Thanks a lot!

**Comment by flexibility2** [comment]
@graykode , hi, sorry to disturb you again, just one more question, I wonder why in the batch_norm layer, it  fixed the learnable parameters, for example:
>conv4 = tf.layers.batch_normalization(conv4, axis=-1, epsilon=1e-5, training=True, **trainable=False**)<br>
Could you give me some explanations? Thanks a lot!
